modules/oxide-components/src/main/ts/keynav/keyboard/Keys.ts

```diff
@@ -1,15 +1,21 @@
 // TODO: This file would need to be updated once we work on https://ephocks.atlassian.net/browse/TINY-8724
 
-const TAB: ReadonlyArray<number> = [ 9 ];
-const ENTER: ReadonlyArray<number> = [ 13 ];
-const SHIFT: ReadonlyArray<number> = [ 16 ];
-const END: ReadonlyArray<number> = [ 35 ];
-const LEFT: ReadonlyArray<number> = [ 37 ];
-const UP: ReadonlyArray<number> = [ 38 ];
-const RIGHT: ReadonlyArray<number> = [ 39 ];
-const DOWN: ReadonlyArray<number> = [ 40 ];
-const SPACE: ReadonlyArray<number> = [ 32 ];
-const ESCAPE: ReadonlyArray<number> = [ 27 ];
+export interface Key {
+  readonly code: string;
+  readonly key: string;
+  readonly which: number;
+}
+
+const TAB: ReadonlyArray<Key> = [{ code: 'Tab', key: 'Tab', which: 9 }];
+const ENTER: ReadonlyArray<Key> = [{ code: 'Enter', key: 'Enter', which: 13 }];
+const SHIFT: ReadonlyArray<Key> = [{ code: 'Shift', key: 'Shift', which: 16 }];
+const END: ReadonlyArray<Key> = [{ code: 'End', key: 'End', which: 35 }];
+const LEFT: ReadonlyArray<Key> = [{ code: 'ArrowLeft', key: 'ArrowLeft', which: 37 }];
+const UP: ReadonlyArray<Key> = [{ code: 'ArrowUp', key: 'ArrowUp', which: 38 }];
+const RIGHT: ReadonlyArray<Key> = [{ code: 'ArrowRight', key: 'ArrowRight', which: 39 }];
+const DOWN: ReadonlyArray<Key> = [{ code: 'ArrowDown', key: 'ArrowDown', which: 40 }];
+const SPACE: ReadonlyArray<Key> = [{ code: 'Space', key: 'Space', which: 32 }];
+const ESCAPE: ReadonlyArray<Key> = [{ code: 'Escape', key: ' ', which: 27 }];
```

Comment:
Why is `key` a space character it should be the key name so `Escape`. If you press escape here you see that it produces a `Escape` when the escape key is pressed. https://w3c.github.io/uievents/tools/key-event-viewer.html

Also not sure we need `code` we `key` should be the right value to check. Code is the physical key regardless of layout. Might not matter much for these keys but it does matter when it comes to letter keys.